spec/features/system/admin/user_management_spec.rb

```diff
@@ -41,7 +41,7 @@
         expect(user_to_change.reload).to be_administrator
       end
 
-      let!(:user_to_delete) { create(:user, name: users.first.name) }
+      let!(:user_to_delete) { create(:user, name: 'A' + users.first.name) }
```

Comment:
If there are any user whose name starts with digits or any character less than `'A'`, this test will fail.